pkg/roachprod/vm/aws/aws.go

```diff
@@ -727,9 +727,26 @@ func (p *Provider) Delete(l *logger.Logger, vms vm.List) error {
 	return g.Wait()
 }
 
-// Reset is part of vm.Provider. It is a no-op.
+// Reset is part of vm.Provider.
 func (p *Provider) Reset(l *logger.Logger, vms vm.List) error {
-	return nil // unimplemented
+	byRegion, err := regionMap(vms)
+	if err != nil {
+		return err
+	}
+	g := errgroup.Group{}
+	for region, list := range byRegion {
+		args := []string{
+			"ec2", "reboot-instances",
+			"--region", region,
+			"--instance-ids",
+		}
+		args = append(args, list.ProviderIDs()...)
+		g.Go(func() error {
+			_, e := p.runCommand(l, args)
```

Comment:
_tiny_ nit: `_, err = p.runCommand(l, args)`

Reply:
Yep, it's a Go convention, and it makes things easier to spot since our eyes have been trained to parse out `err` :)